### PR TITLE
Code quality: Remove nullable reference warnings

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
This pr partially fixes nullable reference warnings. #9895.

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

File: src/Files.App/App.xaml.cs [hunk 1]
```diff
@@ -107,14 +107,14 @@ private IServiceProvider ConfigureServices()
...
-				.AddSingleton<IPreferencesSettingsService, PreferencesSettingsService>((sp) => new PreferencesSettingsService((sp.GetService<IUserSettingsService>() as UserSettingsService).GetSharingContext()))
-				.AddSingleton<IApplicationSettingsService, ApplicationSettingsService>((sp) => new ApplicationSettingsService((sp.GetService<IUserSettingsService>() as UserSettingsService).GetSharingContext()))
-				.AddSingleton<IPaneSettingsService, PaneSettingsService>((sp) => new PaneSettingsService((sp.GetService<IUserSettingsService>() as UserSettingsService).GetSharingContext()))
-				.AddSingleton<ILayoutSettingsService, LayoutSettingsService>((sp) => new LayoutSettingsService((sp.GetService<IUserSettingsService>() as UserSettingsService).GetSharingContext()))
-				.AddSingleton<IAppSettingsService, AppSettingsService>((sp) => new AppSettingsService((sp.GetService<IUserSettingsService>() as UserSettingsService).GetSharingContext()))
+                .AddSingleton<IMultitaskingSettingsService, MultitaskingSettingsService>((sp) => new MultitaskingSettingsService(((UserSettingsService)sp.GetRequiredService<IUserSettingsService>()).GetSharingContext()))
```
Comment (yaichenbaum) [inline src/Files.App/App.xaml.cs:110]: Spacing messed up? Should be set to tabs.

Comment (cinqmilleans) [reply src/Files.App/App.xaml.cs]: The indentation is fixed.

**Review by hez2010 — APPROVED** [review]
LGTM.

**Comment by yaichenbaum** [comment]
@cinqmilleans can you resolve the merge conflicts?

**Comment by yaichenbaum** [comment]
How was this PR validated?

**Comment by hez2010** [comment]
> How was this PR validated?

This PR doesn't have any behavior changes to code. It only helps with reducing nullable warnings during build.

**Comment by cinqmilleans** [comment]
It's up to date. It is assumed that settings services are required. Properties of these instances are called in many places and would cause an error if null. This is not the case for other services.

**Comment by yaichenbaum** [comment]
Thank you

**Pull request merged**